### PR TITLE
refactor(db): Fixes #340 Remove  column createdAt on recoveryCode table

### LINKED ISSUE #340: Remove `createdAt` column on recoveryCode table
These values are never returned to the user and not used internally, so I propose we remove 🗡 the column. Additionally, they are not storing a unix timestamp value.

### PR DESCRIPTION
**Pull request by deeptibaghel**

### CODE REVIEW

**Comment by vbudhram** [comment]
@deeptibaghel Are you sure you ran the migration creation script in https://github.com/mozilla/fxa-auth-db-mysql/issues/340#issuecomment-383588084? It will create a `patch-079-080.sql` and `patch-080-079.sql` migration files.

**Comment by deeptibaghel** [comment]
@vbudhram , ohh i removed the reverse one , will fix it.

File: lib/db/schema/patch-080-079.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
+-- SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+-- alter table recoveryCodes add createdAt BIGINT UNSIGNED NOT NULL;
+
+-- CREATE PROCEDURE `createRecoveryCode_1` (
```
Comment (vbudhram) [inline lib/db/schema/patch-080-079.sql]: These create procedures can be replace with a single drop `DROP PROCEDURE createRecoveryCode_3`. Technically, by dropping this procedure and adding back the column, the database will be in the same state it was before.

**Comment by deeptibaghel** [comment]
@vbudhram thanks, now I have a fair idea about migration process :)

**Comment by vbudhram** [comment]
@jrgm Good point, in hindsight, seems like this migration could have been split into two to avoid that condition. FWIW, I don't expect any user traffic to come until content-server 111.2 is deployed, when mozilla users will see the feature.

**Pull request merged**